src/app/Welcome.js

```diff
@@ -93,7 +93,7 @@ function Info() {
 export default function Welcome(props) {
   const { configs } = props;
   return (
-    <div className="vitessce-container">
+    <div className="vitessce-container vitessce-theme-dark">
```

Comment:
Should this also be pulled from api.js? If Nils decides tomorrow he wants the light theme for the whole demo, can that be a one line change?

(Or maybe not: The priority is the Vitessce component... Sorting out the style rules that are used only in Welcome so they can be either dark or light would be a wasted effort if we never want the whole demo to be light. If this is intentional, add a comment explaining?)

Reply:
Ignore that: api.js will be moving to vitessce-data, and this display detail should stay here, and redeploy is fine.